### PR TITLE
Enable Prettier for `/nodes-base`

### PR DESCRIPTION
**Pull request by ivov**

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -1,4 +1,574 @@
-packages/nodes-base
 packages/editor-ui
 packages/design-system
 *package.json
+
+packages/nodes-base/*
```
Comment (BHesseldieck) [inline .prettierignore:5]: shouldn't this be removed then? or does it just affect "root" level files?

**Review by BHesseldieck — APPROVED** [review]

File: .prettierignore (hunk 1 above)
Comment (ivov) [reply .prettierignore]: You were right, this line was redundant. Removed.

**Review by BHesseldieck — APPROVED** [review]

**Comment by janober** [comment]
Thanks a lot. Got merged.

**Comment by janober** [comment]
Got released with `n8n@0.183.0`

**Pull request merged**